### PR TITLE
[BLOG] Add a-second-czi-grant-for-numpy-and-openblas

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build.](https://labs-git-a-second-czi-grant-for-numpy-and-openblas-quansight.vercel.app/blog/a-second-czi-grant-for-numpy-and-openblas)

## Links to other Quansight blogs:

https://labs.quansight.org/blog/2019/11/numpy-openblas-CZI-grant/ was not updated no llc links found

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

## Links to other Quansight blogs:

### CODE REVIEW

**Comment by MarsBarLee** [comment]
@noatamir 
## Links to other Quansight blogs:

https://labs.quansight.org/blog/2019/11/numpy-openblas-CZI-grant/ was not updated
no llc links found

File: apps/labs/posts/a-second-czi-grant-for-numpy-and-openblas.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+features. Rather, it supports maintenance, community engagement, and other types
+of work that often go unrewarded in open source projects. And although
+[its primary goal is to fund the computational foundations of biology](https://chanzuckerberg.com/eoss/), the benefits of the work done under the Chan
+Zuckerberg Initiative's program extend well beyond specific scientific areas.
+
+[This is the second grant from CZI that NumPy and OpenBLAS have received.](https://labs.quansight.org/blog/2019/11/numpy-openblas-CZI-grant/)
+This past year, the funding has enabled several high-impact improvements to the
+NumPy and OpenBLAS communities. For NumPy, this has meant
+[a new website](https://numpy.org/), the establishment of the
+[Documentation and Website Teams](https://numpy.org/gallery/team.html), and
```
Comment (noatamir) [inline apps/labs/posts/a-second-czi-grant-for-numpy-and-openblas.md:36]:
```suggestion
[Documentation and Website Teams](https://numpy.org/teams/), and
```

Comment (noatamir) [reply apps/labs/posts/a-second-czi-grant-for-numpy-and-openblas.md:36]: link changed

File: apps/labs/posts/a-second-czi-grant-for-numpy-and-openblas.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+certain problem sizes, and the implementation of BLAS extensions already
+provided by similar libraries.
+
+## In practice
+
+As part of the third cycle of the ​Essential Open Source Software for Science
+(EOSS)​ program, CZI awarded $3 million for 17 new grants. These awards bring the
+total number of funded proposals to 67 projects and the EOSS program's total
+commitment to funding scientific open source to $11.8 million. View the
+​[full list of grantees​](https://chanzuckerberg.com/eoss/proposals/).
```
Comment (noatamir) [inline apps/labs/posts/a-second-czi-grant-for-numpy-and-openblas.md:108]:
```suggestion
 ​[full list of grantees​](https://chanzuckerberg.com/eoss/proposals/).
```

**Review by noatamir — APPROVED** [review]
two small changes, otherwise LGTM

**Pull request merged**